README.md

```diff
@@ -62,3 +62,4 @@ We welcome any contributions to the KlickerUZH project. Before considering any c
 The KlickerUZH and all of its subprojects are licensed under the [AGPLv3](https://www.gnu.org/licenses/agpl-3.0.de.html).
 
 [![klicker-uzh](https://img.shields.io/endpoint?url=https://cloud.cypress.io/badge/simple/y436dx/v3&style=for-the-badge&logo=cypress)](https://cloud.cypress.io/projects/y436dx/runs)
+[![Coverage Status](https://coveralls.io/repos/github/uzh-bf/klicker-uzh/badge.svg?branch=v3)](https://coveralls.io/github/uzh-bf/klicker-uzh?branch=v3)
```

Comment:
Should we maybe change the reference branch for this badge to `v3.1-dev` for now and then switch back to v3 once the corresponding release for v3.1 is completed?